src/client/render/render-core.js

```diff
@@ -23,8 +22,8 @@ let renderCore = function() {
 		_previousTime = now;
 
 		// Update active controls, needs to be buttery smooth, thus is called before requesting the next frame
-		if (_controls.enabled === true) {
-			_controls.update(deltaTime);
+		if (renderCore.controls.enabled === true) {
+			renderCore.controls.update(deltaTime);
```

Comment:
If we move this after the `updateCallback`, the tracking camera will use the updated marble positions.